config/confighttp/confighttp.go

```diff
@@ -0,0 +1,48 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package confighttp
+
+import (
+	"net/http"
+	"time"
+
+	"go.opentelemetry.io/collector/config/configtls"
+)
+
+type HttpClientSettings struct {
```

Comment:
Should we break this up into HttpClientSettings and HttpsClientSettings? Or if you don't think we need HttpClientSettings then call this one HttpsClientSettings.

Reply:
What is the difference? I mean only the "insecure" in the TLS config correct?

Reply:
My idea was HTTPClientSettings would have Endpoint and Timeout whereas HTTPSClientSettings would embed HTTPClientSettings but add on TLSSettings.

Reply:
That is not easy to consume by the receiver/exporter. If they want both and let user decide it is hard to use because you don't know which one to embed.